go/oasis-node/cmd/common/common.go

```diff
@@ -200,3 +202,15 @@ func LoadEntity(dataDir string) (*entity.Entity, signature.Signer, error) {
 	entitySignerFactory := fileSigner.NewFactory(dataDir, signature.SignerEntity)
```

Comment:
Actually, why did you introduce a separate `LoadSigner` function? Based on the comment above, the correct way to do this is to add the signer selection part into `LoadEntity`.

And the ledger signer should expose a factory (a `SignerFactory`, see how the file-based signer does it) which only supports the `signature.SignerEntity` role.